docs/web-apps/automated-testing/cypress.md

```diff
@@ -6,184 +6,148 @@ sidebar_label: Cypress
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-This page provides a walkthrough on how to run [Cypress](https://docs.cypress.io/guides/overview/why-cypress.html) tests locally and on Sauce Labs.
+[Cypress](https://docs.cypress.io/guides/overview/why-cypress.html) is a JavaScript front end web application testing framework. `saucectl` -- the Sauce Labs test orchestrator CLI allows you to run Cypress tests directly from your existing Cypress project.
+
+* Don't have Cypress yet? `saucectl` can set up your initial project directory and give you a sample test to run.
+* Already running Cypress? Let `saucectl` run your tests locally in Docker or in the Sauce Labs Cloud, where you have access to thousands of OS/browser combinations and Sauce Labs analytics.  
```

Comment:
According to Kunal, the customers they work with don't really use Docker and they think it's a good idea to put this on the backburner-- not needed to be highlighted in a quickstart